src/constants/chains.ts

```diff
@@ -33,6 +35,7 @@ export const chainInfo: { [chainId in ChainId]: ChainInfo } = {
     cdnUrl: "https://static-sepolia.looksnice.org",
     rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/0xjurassicpunk/looks-distribution",
     cloudinaryUrl: "https://looksrare.mo.cloudinary.net/sepolia",
+    wsUrl: "",
```

Comment:
I guess it's going to be `wss://ws-sepolia.looksrare.org/ws`. Not sure if I should add it now

Reply:
Might as well